src/index.ts

```diff
@@ -1,11 +1,18 @@
-import { Beautifier, Language, BeautifierBeautifyData } from "unibeautify";
+import { Beautifier, Language, BeautifierBeautifyData, DependencyType } from "unibeautify";
 import * as readPkgUp from "read-pkg-up";
 import options from "./options";
 const CLIEngine = require("eslint").CLIEngine;
 const { pkg } = readPkgUp.sync({ cwd: __dirname });
 export const beautifier: Beautifier = {
   name: "ESLint",
   package: pkg,
+  dependencies: [
+    {
+      type: DependencyType.Node,
+      name: "ESLint",
+      package: "eslint",
+    }
+  ],
   options: {
     JavaScript: options.JavaScript,
   },
```

Comment:
Missing change to https://github.com/Unibeautify/beautifier-eslint/blob/dce2b22afd9362491fff0f4d6702318c9ea64cdb/src/index.ts#L21

Something like:

```typescript
  beautify(data: BeautifierBeautifyData) {
    return new Promise<string>((resolve, reject) => {
       const CLIEngine = data.dependencies.get<NodeDependency>("ESLint").package;
```

Reply:
See https://github.com/Unibeautify/unibeautify/blob/master/test/DependencyManager/NodeDependency.spec.ts#L58

Reply:
Whoops, I mean:

```typescript
const { CLIEngine } = data.dependencies.get<NodeDependency>("ESLint").package;
```